scRNA-seq-advanced/README.md

```diff
@@ -0,0 +1,15 @@
+# Advanced scRNA-seq Training Module
+
+This Childhood Cancer Data Lab-designed module covers more advanced topics in the analysis of single-cell RNA-seq data.
+
+The module builds on material in the [scRNA-seq  module](https://github.com/AlexsLemonade/training-modules/tree/master/scRNA-seq), and analyses are designed to be performed within a [Docker container](https://github.com/AlexsLemonade/training-modules/tree/master/docker-install) or on the CCDL RStudio server.
+It covers cell-type identification, integration of multiple single-cell libraries, and differential expression analyses, among other related topics.
```

Comment:
```suggestion
It covers cell-type identification, integration of multiple single-cell RNA-seq libraries, and differential expression analyses, among other related topics.
```

Suggestion to be clear that we're not doing multiomics. Take or leave it, your call!